core/scripts/chaincli/handler/debug.go

```diff
@@ -312,9 +325,35 @@ func (k *Keeper) Debug(ctx context.Context, args []string) {
 	if err != nil {
 		failUnknown("failed to simulate perform upkeep: ", err)
 	}
+	// do tenderly simulations
+	rawCall, err := core.RegistryABI.Pack("simulatePerformUpkeep", upkeepID, performData)
+	if err != nil {
+		failUnknown("failed to pack raw simulatePerformUpkeep call", err)
+	}
+	addLink("simulatePerformUpkeep simulation", tenderlySimLink(k.cfg, chainID, blockNum, rawCall, registryAddress))
+
 	if simulateResult.Success {
```

Comment:
Hey Ryan, what does this Success stand for? I thought true means simulatePerformUpkeep was successful (not reverted). In my test, it returns false, but the simulate [txn](https://dashboard.tenderly.co/leishi/project/simulator/25f70655-5f22-43cc-9862-8a87134c61e4?sharedSimulation=true) is successful and I dont see issues in the forwarder etc (there could be issues, but i didnt see them). Did I miss anything?

Please ignore this messy testing branch/pr. There is issue in latest branch in core repo, has to use this older base.